k8s/vpc-workflow.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   name: vpc-workflow
```

Comment:
Re: `apiVersion: lyra.example.com/v1alpha1`; we should change this to a real URI, probably.